framework/lib/components/combo-picker/combo-picker.tsx

```diff
@@ -78,13 +91,20 @@ export const ComboPicker = ({
       <FormattedNumberInput
         width="100%"
         onChange={ handleInputChange }
-        defaultValue={ inputValue }
+        value={ inputValue }
```

Comment:
Hey what happens if I try to change the value prop from outside, would it react and update the internal state of combo-picker?

Reply:
Fixed